orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -808,16 +808,3 @@ def upsample_nearest2d(g, input, output_size, scale_factors):
 @register_symbolic("upsample_nearest3d")
 def upsample_nearest3d(g, input, output_size, scale_factors):
     return _upsample_nearest(g, input, output_size, scale_factors, "upsample_nearest3d")
-
-
-@register_symbolic("upsample_bilinear2d")
```

Comment:
do you intend to remove aten fallback for this op completely, if so, could you pls also clean up those shape infer logic https://github.com/microsoft/onnxruntime/blob/65575389f4d54ee9f64443160de91d092b8dff90/onnxruntime/python/tools/symbolic_shape_infer.py#L233. And also those in orttraining/orttraining/python/training/ortmodule/_custom_gradient_registry.py

And additionally, since you are introducing ResizeGrad, you might need add such a shape infer logic in the above file also.

Reply:
Not adding the symbolic shape infer for `ResizeGrad` as it is a gradient op and we run the shape inferencing only on the exported forward model (as discussed offline).